### PR TITLE
Add Deploy to AKS example

### PR DESCRIPTION
**Pull request by AbeOmor**
Added a new example for deploying to AKS.

This is a simple example that walks through all the basic steps of deploying to AKS with the R SDK.

I want to add instructions on how to:

- Deploy a Keras Model to AKS
- Attach to an existing AKS Cluster
- Using advanced AKS configurations (Create AKS Cluster in an existing virtual network (optional))

### CODE REVIEW

File: vignettes/deploy-to-aks/deploy-to-aks.Rmd [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+title: "Deploying to AKS with Azure ML SDL for R"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Deploying to AKS with Azure ML SDL for R}
+  %\VignetteEngine{knitr::rmarkdown}
+  \use_package{UTF-8}
+---
+
+This article demonstrates how to deploy a model and conduct inferencing in the model using AzureML SDK for R. We will take a previous saved model. The saved model was trained with the Iris Dataset and can be used to determine if a flower is one of there Iris flower species (setosa, versicolor, virginica).
```
Comment (diondrapeck) [inline vignettes/deploy-to-aks/deploy-to-aks.Rmd]: *three

File: vignettes/deploy-to-aks/deploy-to-aks.Rmd [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+2. Create the deployment environment
+
+3. Create the inferencing configuations
+
+4. Provision an AKS cluster 
+
+5. Deploy the model as a service to the AKS cluster
+
+6. Coduct inferencing on the deployed service.
```
Comment (diondrapeck) [inline vignettes/deploy-to-aks/deploy-to-aks.Rmd]: *Conduct

File: vignettes/deploy-to-aks/deploy-to-aks.Rmd [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+```
+
+Or load an existing workspace from your Azure Machine Learning account
+```{r Initialize a workspace, eval=FALSE}
+ws <- get_workspace("<your workspace name>", "<your subscription ID>", "<your resource group>")
+```
+
+## 1. Register the model
+
+For this example, Register an existing trained model, add name and descirption.
```
Comment (diondrapeck) [inline vignettes/deploy-to-aks/deploy-to-aks.Rmd]: *register and *description

File: vignettes/deploy-to-aks/deploy-to-aks.Rmd [hunk 4]
```diff
@@ -0,0 +1,144 @@
+---
+title: "Deploying to AKS with Azure ML SDL for R"
```
Comment (mx-iao) [inline vignettes/deploy-to-aks/deploy-to-aks.Rmd]: *SDK

File: vignettes/deploy-to-aks/deploy-to-aks.Rmd [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Deploying to AKS with Azure ML SDL for R}
+  %\VignetteEngine{knitr::rmarkdown}
+  \use_package{UTF-8}
+---
+
+This article demonstrates how to deploy a model and conduct inferencing in the model using AzureML SDK for R. We will take a previous saved model. The saved model was trained with the Iris Dataset and can be used to determine if a flower is one of three Iris flower species (setosa, versicolor, virginica).
```
Comment (mx-iao) [inline vignettes/deploy-to-aks/deploy-to-aks.Rmd]: *previously saved model

Comment (mx-iao) [inline vignettes/deploy-to-aks/deploy-to-aks.Rmd]: Can you add a link to whichever sample or vignette has the training code for the model you're using

File: vignettes/deploy-to-aks/deploy-to-aks.Rmd [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+aks_target <- create_aks_compute(ws, cluster_name = 'my-new-cluster')
+wait_for_provisioning_completion(aks_target)
+```
+
+## 5. Deploy the model to AKS and waiting for deployment
+
+Set the web service configuration (using default here)
+
+``` {r Web service configuration, eval=FALSE}
+aks_config <- aks_webservice_deployment_config()
```
Comment (mx-iao) [inline vignettes/deploy-to-aks/deploy-to-aks.Rmd]: can you create aks config with these parameters: `cpu_cores = 1, memory_gb = 1` to showcase using those params. python sdk does the same

**Pull request merged**